packages/supi/src/install/index.ts

```diff
@@ -403,20 +403,32 @@ export async function mutateModules (
       })
     }
 
+    function checkDependencyInPackage (dependencySet: Set<string>, inputs: string[]) {
+      let noneDependencyfound = true
+      for (const input of inputs) {
+        if (!dependencySet.has(input)) {
+          console.log(`No ${input} package found in dependencies of the project`)
```

Comment:
`logger.info` should be used for this

Reply:
and I think it would be nicer to combine these to one message, so just gather the unmatched dependencies to an array and print one info message. Also, there's no need to print the info message if none were matched because in that case the error is thrown